File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.java
// 250723

class Solution {
    public int solution(int[] numbers) {
        int answer = Integer.MIN_VALUE; // 0이 아닌 MIN_VALUE로 선언!
        
        for(int i=0; i<numbers.length; i++) {
            for(int j=0; j<numbers.length; j++) {
                if (i==j) continue;
                if (numbers[i] * numbers[j] > answer) answer = numbers[i] * numbers[j];
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.py
# 260214

def solution(numbers):
    answer = 0
    
    mul_max = numbers[0]*numbers[1]
    for num1 in numbers:
        for num2 in numbers:
            if num2 == num1:
                continue
            else:
                if num1*num2 > mul_max:
                    mul_max = num1*num2
    
    answer = mul_max
    
    return answer